packages/app-types/test/singleExport.test.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { firebase } from '@firebase/app';
```

Comment:
Is this file different than default.d.ts?

Reply:
Yes, I am testing the default export vs. the named `firebase` export. Tested paths are identical but it's a different entrypoint.